src/__test__/redux/actions/projects/saveProject.test.js

```diff
@@ -71,7 +71,7 @@ describe('saveProject action', () => {
     try {
       await store.dispatch(saveProject(mockProject.uuid, mockProject));
     } catch (e) {
-      expect(e).toEqual(errorMsg);
+      expect(e.statusCode).toEqual(400);
     }
```

Comment:
I think this should be

```
expect(async() => {
    await store.dispatch(saveProject(mockProject.uuid, mockProject));
}).rejects.toThrow(APIError)
```

Reply:
no it should not, because all the tests rely on checking the actual error message which is pretty bad because it makes the code very hard to change. In this test I preferred to check the error thrown